README.md

```diff
@@ -17,48 +17,31 @@ Once in a while, it connects to APIs to get the latest information about session
 It then augments and republishes this information for clients to consume. 
 It also provides a couple of extra APIs to save your favorites and accumulate votes.
 
-### Android Application
+### iOS and Android Applications
 
-As you can imagine, the Android version is developed in Kotlin/JVM. What's interesting here is that this time
-application utilizes Multiplatform support, which is an experimental feature in Kotlin 1.3.61. Data structures for retrieving data from the backend server and some date-time operations are shared across multiple projects.
-
-### iOS Application
-
-User interface of iOS version is written in Swift, all logic and data written in Kotlin in the common module. 
-This way iOS part itself is responsible only for specifying how the application looks like and how it represents changes requested by logic. Kotlin and Swift are highly interoperable, so from Swift, you can easily use all classes and tools defined in the common module. 
+Both Android and iOS applications are developed using Kotlin Multiplatform Mobile technology within a single codebase.
+The UI is implemented using Compose Multiplatform library, sharing the same code between platforms.
 
 ## How to build and run
 
-### Building the code
+### Pre-requisites
 
  * Make sure you have the Android SDK installed
```

Comment:
Do we need a Xcode pre-requisite add here?

Reply:
thanks, will add